deployment/k8s/charts/components/bulkLoad/templates/deployment.yaml

```diff
@@ -34,6 +34,15 @@ spec:
           volumeMounts:
             - name: {{ $envMountName }}
               mountPath: /shared
+            {{- if .Values.snapshotVolumeEnabled }}
+            - name: snapshot-volume
+              mountPath: /snapshot
+            {{- end }}
       volumes:
         - name: {{ $envMountName }}
           emptyDir: {}
+        {{- if .Values.snapshotVolumeEnabled }}
```

Comment:
Could you store this value in a local variable?  Maybe that isn't a good idea, but that's the pattern I was following, but I'm happy to stop ;).

Reply:
Changed, open to changing the pattern as well if we don't like it